R/zchunk_LB174.nonghg_bld_USA.R

```diff
@@ -43,7 +43,8 @@ module_gcamusa_LB174.nonghg_bld <- function(command, ...) {
       # Format to level1 sector naming convention
       mutate(sector = gsub("building_", "", GCAM_sector)) %>%
       # using left_join becuase orignal CEDS fuel in NEI has one called "Process", there's no GCAM fuel corresponding to that,
-      # OK to omie, missing values will be dropped later
+      # OK to omit, missing values will be dropped later
+      # TODO: check that the dropped "Process" emissions are not significant
```

Comment:
Can you [enter this as an issue](
https://github.com/JGCRI/gcamdata/issues/new) please? We don't want to depend on TODOs in the code.